cpp/src/io/utilities/file_io_utilities.cpp

```diff
@@ -273,9 +273,18 @@ std::unique_ptr<cufile_input_impl> make_cufile_input(std::string const& filepath
 {
   if (cufile_integration::is_gds_enabled()) {
     try {
-      return std::make_unique<cufile_input_impl>(filepath);
+      auto const cufile_in = std::make_unique<cufile_input_impl>(filepath);
+      CUDF_LOG_INFO("File successfully opened for reading with GDS.");
```

Comment:
Do we need this log? It succeeds, and nothing special here?

Reply:
The goal of the messages in this PR is to provide some transparency into what method is used for file IO. We have direct use of GDS (this branch), no GDS (host IO + memcpy), and kvikIO. I included messages that make this selection apparent. FWIW, this is not the default branch, so very few users/devs will see this message.
I guess we can derive that lack of messages means that GDS is used, but I would prefer to log this case as INFO (off by default). If we find that this is too verbose, I'll remove some of these.